### PR TITLE
Add landing page with the example games

### PR DESCRIPTION
**Pull request by hwo411**
The landing page contains 3 links (out of 5) from Rootlog readme. That would help people who visit the website to get familiar with the tool.

image

Note: I'm unfortunately not an expert in HTML & CSS and it's my first time with Angular. Any suggestions will be more than appreciated.

### CODE REVIEW

**Comment by seiyria** [comment]
things I would like to see:

- mat button for "input your game"
- centered header/subheader
- 3 boxes, 4 wide, 1 for each game (all in one "row" so they flex-wrap), with the label ordering (one per line): winter tournament, game, description, "watch this game" [link to the game])

File: src/app/pages/landing/landing.component.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  <div class="row">
+    <div class="col">
+      <span class="mat-h2">Watch some tournament games:</span>
+    </div>
+  </div>
+  <div class="games">
+    <mat-card class ="game" *ngFor="let game of games"><mat-card-title>{{game.title}}</mat-card-title><mat-card-subtitle>{{game.subtitle}}</mat-card-subtitle><a mat-raised-button color="accent" href="/full-game?gameUrl={{game.url}}">Watch</a></mat-card>
+  </div>
+    <div class="row" style="justify-content: center">
```
Comment (seiyria) [inline src/app/pages/landing/landing.component.html:21]: please don't use `style` - use a CSS class

File: src/app/pages/landing/landing.component.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col">
+      <span class="mat-h2">Watch some tournament games:</span>
+    </div>
+  </div>
+  <div class="games">
+    <mat-card class ="game" *ngFor="let game of games"><mat-card-title>{{game.title}}</mat-card-title><mat-card-subtitle>{{game.subtitle}}</mat-card-subtitle><a mat-raised-button color="accent" href="/full-game?gameUrl={{game.url}}">Watch</a></mat-card>
```
Comment (seiyria) [inline src/app/pages/landing/landing.component.html:19]: the = is spaced out a bit far

File: src/app/pages/landing/landing.component.scss [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+a.link {
+  color: pink;
+}
+
+a:visited.link {
+  color: violet;
+}
+
+.games {
+  display:  flex;
```
Comment (seiyria) [inline src/app/pages/landing/landing.component.scss:11]: extra space

**Review by seiyria — APPROVED** [review]

File: src/app/pages/landing/landing.component.html (hunk 1 above)
Comment (hwo411) [reply src/app/pages/landing/landing.component.html:21]: Ah, forgot to add a style, thanks for pointing

**Pull request merged**